src/posts/what-is-wai.md

```diff
@@ -0,0 +1,108 @@
+
+WAI brings people from disability organizations, governments and research labs from all around the world to develop guidelines and resources to make the Web accessible to people with disabilities that include auditory, cognitive, neurological, physical, speech and visual disabilities.
+
+In short, WAI develops accessibility standards and offers implementation guidance.
+
+As described by [W3C Website](https://www.w3.org/WAI/about/), WAI “pursues accessibility of the web through these primary activities”:
+
+- ensuring that W3C standards support accessibility
+- developing accessibility guidelines for web content and applications,
+  browsers, and authoring tools
+- developing resources to improve web accessibility evaluation
+  processes and tools
+- supporting education and outreach on web accessibility
+- coordinating with research and development that may impact future  
+  accessibility of the web
+- promoting harmonized international uptake of web accessibility  
+  standards
+
+Two best examples of WAI’s significance in everyday technology life are the Web Content Accessibility Guidelines, (WCAG) and Accessible Rich Internet Applications (WAI-Aria) Recommendations that were created by members of WAI.
+
+## Web Content Accessibility Guidelines, (WCAG) and Accessible Rich Internet Applications (WAI-Aria) Recommendations:
+
+WAI’s effort created guidelines for developers, designers and people in tech to use every single day. Some of the most used guidelines and applications have been WCAG and WAI-ARIA.
+
+[Web Content Accessibility Guidelines](https://www.w3.org/WAI/standards-guidelines/wcag/), or WCAG is a benchmark for web accessibility. WCAG was created with a goal to clearly define technical standards and unifying accessibility standards. WCAG’s standards are categorized under 4 principles.
+
+- Perceivable:
+
+  Users should be able to identify content and elements by means of senses. For example, if a website's navigation changes from page to page, users will have to relearn the navigation of the site, which will not be a great user experience.
+
+- Operarable:
+
+  Users should successfully use controls, buttons, navigation, and other interactive elements on the page. One example of this would be that websites should be operable by a keyboard and the page should have the same functionalities it has when used by a mouse to navigate.
+
+- Understandable
+
+  Users should be able to comprehend the content and learn from your website. One example of this would be if abbreviations or jargons are used, they should be defined for the user.
+
+- Robust:
+
+  Websites should be able to be accessed by a wide variety of users and technologies. For example, websites should be accessible for screen readers and many different browser versions.
+
+Each WCAG guideline has success criteria broken down to three standards, A, AA, AAA, with triple A being the standard we should aim for.
+
```

Comment:
It feels like we should have a heading here, as we transition from talking about the WCAG to ARIA.